tests/pdctl/log/log_test.go

```diff
@@ -33,64 +33,110 @@ func Test(t *testing.T) {
 
 var _ = Suite(&logTestSuite{})
```

Comment:
If the two test cases run concurrently, is there a possibility that this test is unstable?

Reply:
These two cases won't run concurrently.